thoth/package_extract/core.py

```diff
@@ -69,10 +69,11 @@ def extract_image(image_name: str, timeout: int = None, *,
         result = run_analyzers(rootfs_path)
         result['layers'] = layers
 
-    push_gateway = os.getenv('PROMETHEUS_PUSH_GATEWAY', 'pushgateway:9091')
-    if push_gateway:
+    push_gateway_host = os.getenv('PROMETHEUS_PUSHGATEWAY_HOST')
+    push_gateway_port = os.getevn('PROMETHEUS_PUSHGATEWAY_PORT')
+    if push_gateway_host:
```

Comment:
what if PROMETHEUS_PUSHGATEWAY_PORT is not set and push_gateway_port will be None?!